Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -255,7 +269,8 @@ class HistoryPanelSiteTableViewController: SiteTableViewController {
     @objc func refresh() {
         self.refreshControl?.beginRefreshing()
         resyncHistory()
-        historyPanel?.updateNumberOfSyncedDevices()
+        historyPanel?.fetchSyncedDevicesCount()
```

Comment:
`fetchSyncedDevicesCount` is async so the below method may not have the updated count. You'll need to chain the two using `uponQueue`.

You also don't need this since `resyncHistory` calls these after it's sync it's history [1]

[1] https://github.com/mozilla/firefox-ios/pull/1874/files#diff-acde690772668c1259217314efbefe8cR239